README.md

```diff
@@ -197,50 +228,112 @@ TableName user1
 
 > Tips: The node's method name can be found in the Listener file under the corresponding SQL directory
 
-### Clean
-
-Clear the **comments** and **spaces** before and after
+<br/>
 
+### Split sql by statement
+Take `FlinkSQL` as an example:
 ```javascript
-import { cleanSql } from 'dt-sql-parser';
-
-const sql = `-- comment comment
-select id,name from user1; `
-const cleanedSql = cleanSql(sql)
-console.log(cleanedSql)
+import { FlinkSQL } from 'dt-sql-parser';
+const parser = new FlinkSQL();
+const sql = 'SHOW TABLES;\nSELECT * FROM tb;';
+const sqlSlices = parser.splitSQLByStatement(sql);
+console.log(sqlSlices)
 
 /*
-select id,name from user1;
+[
+    {
+    startIndex: 0,
+    endIndex: 11,
+    startLine: 1,
+    endLine: 1,
+    startColumn: 1,
+    endColumn: 12,
+    text: 'SHOW TABLES;'
+    },
+    {
+    startIndex: 13,
+    endIndex: 29,
+    startLine: 2,
+    endLine: 2,
+    startColumn: 1,
+    endColumn: 17,
+    text: 'SELECT * FROM tb;'
+    }
+]
 */
-```
-
-### Split SQL
-
-When the SQL text is very big, you can think about to split it by `;` , and handle it by each line.
 
-```javascript
-import { splitSql } from 'dt-sql-parser';
-
-const sql = `select id,name from user1;
-select id,name from user2;`
-const sqlList = splitSql(sql)
-console.log(sqlList)
-
-/*
-["select id,name from user1;", "\nselect id,name from user2;"]
-*/
 ```
 
-### Other API
-
-- parserTreeToString(input: string)
+<br/>
+
+### Auto Complete
+Get the autocomplete information in the specified position of sql, using `FlinkSQL` as an example:
```

Comment:
Obtaining autocomplete information at a specified position in SQL.
we can refer to the example of using `FlinkSQL`.